exchanges/bittrex/bittrex_wrapper.go

```diff
@@ -711,12 +916,80 @@ func (b *Bittrex) ValidateCredentials(assetType asset.Item) error {
 	return b.CheckTransientError(err)
 }
 
+// FormatExchangeKlineInterval returns Interval to string
+// Overrides Base function
+func (b *Bittrex) FormatExchangeKlineInterval(in kline.Interval) string {
+	switch in {
+	case kline.OneMin:
+		return "MINUTE_1"
+	case kline.FiveMin:
+		return "MINUTE_5"
+	case kline.OneHour:
+		return "HOUR_1"
+	case kline.OneDay:
+		return "DAY_1"
+	default:
+		return "notfound"
+	}
+}
+
 // GetHistoricCandles returns candles between a time period for a set time interval
+// Candles set size returned by Bittrex depends on interval length:
+// - 1m interval: candles for 1 day (0:00 - 23:59)
+// - 5m interval: candles for 1 day (0:00 - 23:55)
+// - 1 hour interval: candles for 31 days
+// - 1 day interval: candles for 366 days
+// This implementation rounds returns candles up to the next interval or to the end
+// time (whichever comes first)
 func (b *Bittrex) GetHistoricCandles(pair currency.Pair, a asset.Item, start, end time.Time, interval kline.Interval) (kline.Item, error) {
```

Comment:
The data Bittrex requires for this function and the data they return is quite odd compared to other exchanges. They don't have a end date system. So you must specify a start date and take all the data they provide (if I understand correctly). You might have to add in several if conditions to ensure that the exchange requirements are satisfied so that we don't get errors. This is one of the errors I received.
![image](https://user-images.githubusercontent.com/31364354/113067771-b7719500-9208-11eb-87d3-feb71383a84c.png)

Reply:
I understand I'm being fairly picky with this one since if I input correct values it works. However since we support a default value but the exchange does not, this might be fairly confusing for a new user. So for that reason it would be awesome if you could please make this change and also add a test case for it.
![image](https://user-images.githubusercontent.com/31364354/113068321-d45a9800-9209-11eb-81bc-5b69165a6ed4.png)

Reply:
OK, I'll give it a try!
Thanks :)

Reply:
@MadCozBadd : I think it should be OK now, could you please check?

Reply:
Perfect!!!!